drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -345,7 +345,7 @@ static int _atwinc15x0_get(netdev_t *netdev, netopt_t opt, void *val,
     assert(dev == atwinc15x0);
 
     DEBUG("%s dev=%p opt=%u val=%p max_len=%u\n", __func__,
-          netdev, opt, val, max_len);
+          (void*)(uintptr_t)netdev, opt, val, max_len);
```

Comment:
```suggestion
          (void *)netdev, opt, val, max_len);
```

But here a data pointer is casted to a data pointer. I don't think that two casts in a row improve readability.